examples/libxml2/libxml2_example_test.cc

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <dirent.h>
+#include <memory>
+#include "gtest/gtest.h"
+
+namespace {
+
+size_t CountFilesInDir(const std::string& path) {
+  size_t res = 0;
+  std::unique_ptr<DIR, decltype(&closedir)> dir(opendir(path.c_str()),
+                                                &closedir);
+  if (!dir) return res;
+  while (readdir(dir.get())) {
+    ++res;
+  }
+  return res;
+}
+
+}  // namespace
+
+TEST(LibXml2ExampleTest, Crash) {
```

Comment:
I'd rather keep fewer dependencies, such small lib does not deserve multiple testing frameworks